integration-tests/build.gradle

```diff
@@ -51,6 +56,8 @@ subprojects {
         doFirst {
             dockerCompose.solr.exposeAsSystemProperties(integrationTest)
             systemProperty 'solrFlavor', "${solrFlavor}"
+            systemProperty 'edition', edition
```

Comment:
Can you make this property `alfrescoEdition` (both the property name and the variable name) to make it more clear what it's about?

Reply:
No problem, I changed it.